rules/integrations/endpoint/elastic_endpoint_security_behavior_detected.toml

```diff
@@ -0,0 +1,75 @@
+[metadata]
+creation_date = "2024/03/24"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2024/03/24"
+promotion = true
+
+[rule]
+author = ["Elastic"]
+description = """
+Generates a detection alert each time an Elastic Endpoint Security alert for malicious behavior is received. Enabling this rule allows you to
+immediately begin investigating your Endpoint behavior alerts. This rule identifies Endpoint Security behavior detections only, and does not include prevention alerts.
+"""
+enabled = true
+from = "now-10m"
+index = ["logs-endpoint.alerts-*"]
+language = "kuery"
+license = "Elastic License v2"
+max_signals = 10000
+name = "Behavior - Detected - Endpoint Security"
```

Comment:
current naming convention is `Elastic Defend` vs old `Elastic Endpoint Security`